### PR TITLE
Update pastebin meta

### PR DESCRIPTION
**Pull request by krishnasism**

### CODE REVIEW

File: modules/sfp_pastebin.py [hunk 1]
```diff
@@ -20,12 +20,32 @@ class sfp_pastebin(SpiderFootPlugin):
...
+            'model': "FREE_AUTH_LIMITED",
+            'references': [
+                "https://pastebin.com/doc_api",
+                "https://pastebin.com/faq"
+            ],
+            'apiKeyInstructions': [
+                "Visit pastebin.com",
+                "Register a free account",
+                "Navigate to pastebin.com/doc_api",
+                "The API key is listed under 'Your Unique Developer API Key'"
```
Comment (smicallef) [inline modules/sfp_pastebin.py:39]: Actually it's the Google search API they need to obtain.

Comment (krishnasism) [reply modules/sfp_pastebin.py:39]: So, instructions for getting google custom search api?

Comment (smicallef) [reply modules/sfp_pastebin.py:39]: Yep

Comment (krishnasism) [reply modules/sfp_pastebin.py]: Updated

**Pull request merged**